cfme/tests/infrastructure/test_advanced_search_providers.py

```diff
@@ -19,25 +16,8 @@
     is_cfme_exception, cfme_exception_text)
 
 
-pytestmark = [pytest.mark.usefixtures("setup_cleanup_search"), pytest.mark.tier(3)]
-
-
-@pytest.fixture(scope="module")
-def single_provider():
-    """Ensure the infra provider is setup"""
-    try:
-        return setup_a_provider_by_class(InfraProvider)
-    except Exception as ex:
-        pytest.skip("Exception while setting up providers, therefore skipping: {}".format(ex))
-
-
-@pytest.fixture(scope="module")
-def hosts_with_vm_count(hosts):
```

Comment:
Yep, good eye! If you take a look at `test_advanced_search_hosts.py`, you will find it there - and it's used. This is a copy-paste from there that was not used anywhere in this module so I got rid of it.